src/agor/tools/agent_handoffs.py

```
@@ -286,57 +501,199 @@ def generate_mandatory_session_end_prompt(
 
 
 def generate_meta_feedback(
```

Pylint (3.3.7)

[refactor] Too many arguments (9/5)

(R0913)